src/mac.cpp

```diff
@@ -354,30 +356,32 @@ std::vector<std::string> GetClickableButtons() {
   return result;
 }
 
-std::tuple<std::string, int> GetEditorState(bool fallback) {
-  std::tuple<std::string, int> result;
-  std::get<0>(result) = "";
-  std::get<1>(result) = 0;
+std::tuple<std::string, int, bool> GetEditorState() {
+  std::tuple<std::string, int, bool> result;
+  std::get<2>(result) = true;
```

Comment:
Should the first two entries of result be initialized as well? A similar pattern shows up in the Linux version too.

Reply:
I checked with Matt and realized that the initialization was unnecessary, since strings will initialize to `""` and integers will be `0`. `bool` will end up being `false`, though, so have to initialize that one.